src/ZfBugsnag/Service/BugsnagService.php

```diff
@@ -67,33 +67,49 @@ public function setupErrorHandlers()
      */
     public function errorHandler($errno, $errstr, $errfile = '', $errline = 0, $errcontext = [])
     {
-        $result = $this->bugsnag->errorHandler($errno, $errstr, $errfile, $errline);
+        try
+        {
+            $this->bugsnag->errorHandler($errno, $errstr, $errfile, $errline);
+        } catch (\Exception $e) {
+            // Something wrong in the bugsnag notify call
```

Comment:
I wasn't sure what to do here.  In our own projects we typically do an `error_log` in these situations so that at least we can get some notification that bugsnag is failing.  However, in general purpose library code I wasn't sure that was the best option.  Any thoughts?